### PR TITLE
Push the actual content items to the top of sidebar

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by mit-mit** [comment]
@sethladd @LarkAscending

**Comment by mit-mit** [comment]
@sethladd @LarkAscending any feedback? This is a small incremental step unrelated to the larger step we are discussing elsewhere.

**Comment by InMatrix** [comment]
I think it's reasonable to temporarily downplay the "Flutter examples" link until we have a real examples page instead of a GitHub repo.

File: _includes/sidebar.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
       <li>
         <a href="{{folder.external_url}}" target="_blank">{{ folder.title }}</a>
       </li>
+      {% elsif folder.spacer %}
+      <br>
```
Comment (mit-mit) [inline _includes/sidebar.html]: I'm unfortunately pretty html/css illiterate :-). Got a tip for what that would look like?

Comment (sethladd) [inline _includes/sidebar.html]: I'll take a crack at it. Would you mind opening an issue for me? You can merge this and assign the other issue to me. Thanks!

**Pull request merged**